packages/kuma-gui/src/app/connections/data/index.ts

```diff
@@ -4,7 +4,7 @@ const appProtocols = ['http', 'tcp', 'grpc'] as const
 // `_` followed by 1 to 5 digits followed by a `.`
 const trailingPortRe = /_\d{1,5}\./
 const trailingPortRe2 = /_\d{1,5}/
-const meshServiceRe = /_(mz|m|ext){1}svc_\d{1,5}$/
+const meshServiceRe = /_(mz|m|ext){1}svc_\d{1,5}/
```

Comment:
```suggestion
const meshServiceRe = /_(mz|m|ext){1}svc_\d{1,5}(-[a-z0-9]+)?$/
```

I think this would be the very correct version of the regex

Reply:
Ah even better 🎉  thanks!